### ISSUE TITLE
tabIndex cannot be dynamic

### ISSUE THREAD

**Issue by timtyrrell** [evcohen/eslint-plugin-jsx-a11y#50]
It is seems really unfortunate that you can't pass a tabIndex in as a parameter to an element, like:

<a onClick={ something() } tabIndex={ dynamicIndex } />

This triggers the "onclick-has-focus" rule. Do you have any thoughts on that?

**Comment by ljharb** [comment]
@timtyrrell we can't see your example - can you enclose it in single or triple backticks?

**Comment by timtyrrell** [comment]
Doh, my bad. Like if I am building a list of links, I want to send in a different index to each. I am knew at the a11y stuff so tell me if I am off-base.
```
<a onClick={ something() } tabIndex={ dynamicIndex } />
```

**Comment by ljharb** [comment]
well, you're passing the *return value* of `something` to `onClick`, not `something` itself - is that return value a function?

**Comment by timtyrrell** [comment]
I am sorry to distract you, that was a totally fake part, I was only focused on the tabIndex piece:
```
<a onClick={ handleClick } tabIndex={ dynamicIndex } />
```

**Comment by timtyrrell** [comment]
It is checking for a number, I just wondered if anyone had thoughts on if I am handling anything incorrectly

https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/src/rules/onclick-has-focus.js#L34

**Comment by evcohen** [comment]
you're right, it shouldn't warn in that case. fixing now, thanks!

**Comment by evcohen** [comment]
This is fixed in [v1.2.3](https://github.com/evcohen/eslint-plugin-jsx-a11y/releases/tag/1.2.3)

**Comment by timtyrrell** [comment]
@evcohen Sorry for the late "thanks!". On my todo list to check out still
